docs/guide/building.md

```diff
@@ -38,6 +38,8 @@ format , builds ES6 version
 * `npm run bundle:typings` - generates TS typing, only emits
 ‘.d.ts’ declaration files
 
+Note: We recommend using `node@16.14.0` for building HF. Other versions are not supported.
```

Comment:
Nowhere in the docs, except for some code comments, we refer to the product at "HF". Let's use the full name.

Also, even after this change, our current CI setup uses the latest version of Node 16, so specifying such an exact version seems odd.

I propose to change this text to:

We use the latest version of Node 16 LTS in the build-chain and recommend this version for building. Note that for using (not building) HyperFormula, a wider range of Node versions is supported.

Reply:
done